src/main/resources/com/uber/jenkins/phabricator/PhabricatorNotifier/global.jelly

```diff
@@ -11,10 +11,12 @@
     tags they use. Views are always organized according to its owner class,
     so it should be straightforward to find them.
   -->
-  <f:section title="Uberalls" name="uberalls">
-    <f:entry title="Uberalls URL" field="uberallsURL"
-      description="Location of the Uberalls instance">
-      <f:textbox />
-    </f:entry>
-  </f:section>
+  <f:section title="Uberalls">
+    <f:optionalBlock title="Arc Callback with Uberalls" name="uberalls" checked="true">
```

Comment:
I like this. Note to self: ensure that existing jobs get auto-checked when pulling this change in.